packages/appcd-telemetry/src/telemetry.js

```diff
@@ -42,9 +42,9 @@ export default class Telemetry extends Dispatcher {
 			throw new TypeError('Expected config to be a valid config object');
 		}
 
-		const aguid = cfg.get('telemetry.guid');
-		if (!aguid || typeof aguid !== 'string') {
-			throw new Error('Config is missing a required, valid "telemetry.guid"');
+		const app = cfg.get('telemetry.guid') || cfg.get('telemetry.app');
```

Comment:
We should favor `telemetry.app` over `telemetry.guid`. I don't see how this would affect backwards compatibility.

Reply:
If someone has already set `telemetry.guid`, it'll start being ignored in the future if that's not first. It might be that we don't care about that, but it's technically breaking.

Reply:
I don't believe anybody is using appcd-telemetry other than the daemon. `guid` will only be ignored if `app` is set, which is what consumers should be using anyways. The problem is if someone has `app` and `guid` set at the same time, which shouldn't happen, in which case we want them to use `app`, right?